website/docs/concepts/html/events.mdx

```diff
@@ -694,3 +694,7 @@ which will remove the event listener from the element.
 
 For more information on `EventListener`, see the
 [gloo_events docs.rs](https://docs.rs/gloo-events/0.1.1/gloo_events/struct.EventListener.html).
+
+## Using EventListener in a Function Component
+
+See [Applying event listeners to the DOM](./function-components/hooks/use-effect#applying-event-listeners-to-the-dom).
```

Comment:
```suggestion
See [Applying event listeners to the DOM](../function-components/hooks/use-effect#applying-event-listeners-to-the-dom).
```
Can you make a new PR with this change? The link is broken right now

Reply:
Thanks for noticing. I created the PR. How can I check the rendered output?

https://github.com/yewstack/yew/pull/2491